src/swap/SwapReviewScreen.tsx

```diff
@@ -64,6 +64,8 @@ export function SwapReviewScreen() {
   const walletAddress = useSelector(walletAddressSelector)
   const celoAddress = useSelector(celoAddressSelector)
   const feeCurrency = useFeeCurrency() ?? celoAddress
+  const [quoteRequestAt, setQuoteRequestAt] = useState<number | undefined>()
+  const [quoteResponseAt, setQuoteResponseAt] = useState<number | undefined>()
```

Comment:
setting new state triggers a rerender. since that's not really needed here, should we use a `useRef` instead of a `useState`?